src/App.js

```diff
@@ -34,7 +35,10 @@ class App extends Component {
           <Route exact path="/flexicon/InternalContactOverview" component={ InternalContactOverview } />
           <Route exact path="/flexicon/orders" component={ OrdersPage }/>
           <Route exact path="/flexicon/customers" component={ CustomerList } />
+          <Route exact path="/flexicon/orders/:orderNumber" component={ OrderDetails }/>
```

Comment:
why do you have this here? We have the route to orders ..

Reply:
I will see... this is what I pulled from master. Did not add it myself... will delete it again.
:smile:

Reply:
:+1: